skate-plugin/src/main/kotlin/com/slack/sgp/intellij/ChangelogParser.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2023 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.slack.sgp.intellij
+
+import java.time.LocalDate
+
+private val LOCAL_DATE_REGEX = "^\\d{4}-\\d{2}-\\d{2}\$".toRegex()
+private val String.isLocalDate: Boolean
+  get() {
+    return LOCAL_DATE_REGEX.matches(this)
+  }
+
+object ChangelogParser {
```

Comment:
Let's add a doc to this

Reply:
Doc added: 37f1b004893743472769c57b51e495f7f8c109da

Reply:
I think it landed in the wrong place? I don't see it on `ChangelogParser`

Reply:
I added a doc to `ChangelogParser.kt`